crates/lib/tests/serde.rs

```diff
@@ -58,25 +58,27 @@ fn test_json_mappings() {
         ("and_peggy", AlgebraicType::option(AlgebraicType::F64)),
         ("identity", Identity::get_type()),
     ]);
+
     let data = r#"
 {
     "foo": 42,
     "bar": "404040FFFF0A48656C6C6F",
     "baz": ["heyyyyyy", "hooo"],
     "quux": { "Hash": "54a3e6d2b0959deaacf102292b1cbd6fcbb8cf237f73306e27ed82c3153878aa" },
     "and_peggy": { "some": 3.141592653589793238426 },
-    "identity": ["0000000000000000000000000000000000000000000000000000000000000000"]
+    "identity": ["0x0"]
```

Comment:
Ah the Rust implementation probably needs to revert to original hex stringification as well. We do use identity/address strings in a few places (e.g. configs).